go-controller/pkg/config/config.go

```diff
@@ -216,6 +216,9 @@ type DefaultConfig struct {
 	// that are initiated from the pods so that the reverse connections go back to the pods.
 	// This represents the conntrack zone used for the conntrack flow rules.
 	ConntrackZone int `gcfg:"conntrack-zone"`
+	// DisableRequestedChassis flag to indicate if logical_switch_port requested-chassis
+	// options can be disabled
```

Comment:
maybe a comment here as to why this is helpful for dpu full mode? or under what circumstance you want to enable this flag?